yaas/parsers/blueprint.py

```diff
@@ -36,8 +36,8 @@ def add(client, args):
             raw_blueprint = sys.stdin.read()
 
         client.blueprint.add(
-            blueprint=json.loads(raw_blueprint),
             blueprint_name=args.name,
+            blueprint=json.loads(raw_blueprint),
```

Comment:
Just FYI, this change wasn't necessary (but I like it if we decide to keep the new ordering).